pkg/exporter/exporter.go

```diff
@@ -0,0 +1,24 @@
+package exporter
```

Comment:
make exporter design consistent with collector and diagnoser as there is no reason for it to be different. This allows new exporters to be defined using the same pattern as collector / diagnose